### PR TITLE
Update jQuery ajax header Authorization setting

### PR DESCRIPTION
**Pull request by thang12l**
Correct Authorization Bearer setting for jQuery ajax header

### CODE REVIEW

**Comment by squatto** [comment]
@thang12l doesn't doing it the way that you're proposing overwrite any existing custom headers? We changed it to directly set the "Authorization" value in the `headers` array to avoid that.

**Comment by thang12l** [comment]
@squatto the proposing way will overwrite custom headers if they exist. Just update it again to avoid it. When no headers exist, use ajaxSetup to add the new setting.

File: src/ShopifyApp/resources/views/partials/token_handler.blade.php [hunk 1]
```diff
@@ -23,7 +23,13 @@
 
         if (window.jQuery) {
             // jQuery
-            window.jQuery.ajaxSettings.headers['Authorization'] = `Bearer ${window.sessionToken}`;
+            if (window.jQuery.ajaxSettings.headers) {
+                window.jQuery.ajaxSettings.headers['Authorization'] = `Bearer ${window.sessionToken}`;
+            } else {
+                window.jQuery.ajaxSetup({
+                    headers: { 'Authorization': 'Bearer ' + window.sessionToken}
```
Comment (osiset) [inline src/ShopifyApp/resources/views/partials/token_handler.blade.php:30]: @thang12l Can you adjust this line to use template syntax like above :) I can merge then.

**Review by thang12l — COMMENTED** [review]
Just updated with the template style

**Comment by osiset** [comment]
Thanks! I havent used jQuery in a long time, but if `window.jQuery.ajaxSettings.headers` is `undefined`, can we just `window.jQuery.ajaxSettings.headers = {};` ? Or does that not work, as in, you have to previously have used ajaxSetup?

**Comment by thang12l** [comment]
@osiset window.jQuery.ajaxSettings.headers = {}; will work like a charm!

**Comment by osiset** [comment]
@thang12l Nice! Update your PR and ill merge it in.

**Comment by thang12l** [comment]
@osiset , Already done with 7a0aa37c10f4089590d9286e3cc21b7b6a43b3a6. Please merge it in.

**Pull request merged**